proto/proto/requests.proto

```diff
@@ -31,14 +31,37 @@ message GetBlockHeaderByNumberRequest {
 }
 
 // State synchronization request.
+//
+// Specifies state updates the client is intersted in. The server will return
+// the first block which contains a note matching `note_tags` or the chain tip.
+// And the corresponding updates to `nullifiers` and `account_ids` for that
+// block range.
```

Comment:
Awesome comments!

minor nit: we usually wrap lines after 100 chars. Is there a reason to wrap them after 80 chars here?

Reply:
update `*.proto` to 100 chars